### PR TITLE
[viewer] bi(callbacks): onExpand image & gallery

### PR DESCRIPTION
**Pull request by Barackos**

### CODE REVIEW

**Comment by mharsat** [comment]
Just have in mind that the `helpers` prop is on its way to deprecation

File: examples/main/shared/viewer/Viewer.jsx [hunk 1]
```diff
@@ -49,6 +49,8 @@ export default class Viewer extends PureComponent {
         expandModeIndex: this.expandModeData.imageMap[entityIndex] + innerIndex,
       });
     },
+    viewerAction: async (pluginId, actionName) =>
```
Comment (lxgreen) [inline examples/main/shared/viewer/Viewer.jsx:52]: eeh.. poorly named
should be a function name

File: packages/plugin-gallery/web/src/gallery-viewer.jsx [hunk 2]
```diff
@@ -144,8 +145,9 @@ class GalleryViewer extends React.Component {
   };
 
   handleExpand = data => {
-    const { onExpand } = this.props.helpers;
-    onExpand && onExpand(this.props.entityIndex, data.idx);
+    const { onExpand, viewerAction } = this.props.helpers;
+    viewerAction?.(GALLERY_TYPE, 'expand_image');
```
Comment (lxgreen) [inline packages/plugin-gallery/web/src/gallery-viewer.jsx:149]: here you can see how bad is the name. tells literally nothing

File: packages/plugin-image/web/src/image-viewer.jsx [hunk 3]
```diff
@@ -212,8 +213,9 @@ class ImageViewer extends React.Component {
 
   handleExpand = e => {
     e.preventDefault();
-    const { onExpand } = this.props.helpers;
-    onExpand && onExpand(this.props.entityIndex);
+    const { onExpand, viewerAction } = this.props.helpers;
+    viewerAction?.(IMAGE_TYPE, 'expand_image');
```
Comment (lxgreen) [inline packages/plugin-image/web/src/image-viewer.jsx:217]: also here.

**Comment by Barackos** [comment]
Squashed into a new PR, #1093

**Pull request closed without merging**